### PR TITLE
Fix broken link in DART tutorial

### LINKED ISSUE #441: bug: Broken link in the DART Tutorial for Configuring MATLAB for DART and netCDF
### Describe the bug

1. List the steps someone needs to take to reproduce the bug.

DART user Anthony Holmes reported to Brett @braczka that there's a broken link on the DART Tutorial page for Configuring Matlab® for netCDF & DART:

https://raw.githubusercontent.com/NCAR/DART/v10.6.0/theory/readme.rst

http://www.image.ucar.edu/DAReS/DART/DART2_Documentation.php#configure_matlab

2. What was the expected outcome?

The user should be linked to a description of how to add a path to MATLAB so that the DART MATLAB scripts are accessible.

3. What actually happened? 

The following error message is displayed.

### Error Message  

**Forbidden** You don't have permission to access this resource.

### Which model(s) are you working with?

None, this is in the DART Tutorial Documentation.

### Screenshots   

image

### Version of DART      
v10.6.0.

### Have you modified the DART code?   
No.

### Build information
Please describe:  
 1. Computer you are using: Macbook Pro
 2. The compiler you are using: N/A

### PR DESCRIPTION
**Pull request by johnsonbk**
## Description:

There's a broken link on the DART Tutorial page for Configuring Matlab® for netCDF & DART:

https://raw.githubusercontent.com/NCAR/DART/v10.6.0/theory/readme.rst

### Fixes issue

fixes #441  

### Types of changes

- [X] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [X] Documentation update

### Documentation changes needed?

- [X] My change requires a change to the documentation.
  - [X] I have updated the documentation accordingly.

### Tests

I rebuilt the documentation using Sphinx to check that the two edited pages render correctly.

## Checklist for merging

- [x] Updated changelog entry
- [X] Documentation updated
- [X] Update conf.py

## Checklist for release
- [ ] Merge into main
- [ ] Create release from the main branch with appropriate tag
- [ ] Delete feature-branch

## Testing Datasets

- [ ] Dataset needed for testing available upon request
- [ ] Dataset download instructions included
- [X] No dataset needed

### CODE REVIEW

**Comment by hkershaw-brown** [comment]
Don't merge this yet, will bundle with #440

File: guide/matlab-observation-space.rst [hunk 1]
```diff
@@ -1,9 +1,27 @@
...
+To allow your environment to seamlessly use the DART MATLAB functions, your
+MATLAB path must be set to include two of the directories in the DART
+repository. In the MATLAB command prompt enter the following, using the real
+path to your DART installation:
+
+.. code-block::
+
+   addpath('DART/diagnostics/matlab','-BEGIN')
+   addpath('DART/guide/DART_LAB/matlab','-BEGIN')
+
```
Comment (braczka) [inline guide/matlab-observation-space.rst:20]: ```It's very convenient to put these commands in your ~/matlab/startup.m so they get run every time MATLAB® starts up. DART provides an example diagnostics/matlab/startup.m that you can use. It is internally documented.```

Comment (braczka) [reply guide/matlab-observation-space.rst:20]: I think we want to add this too -- right?

**Review by braczka — CHANGES_REQUESTED** [review]
I think we want to make mention of the startup.m file.....

File: guide/matlab-observation-space.rst (hunk 1 above)
Comment (johnsonbk) [reply guide/matlab-observation-space.rst:20]: You're right. I added those instructions.

**Review by braczka — APPROVED** [review]

**Pull request merged**